packages/pwa-kit-react-sdk/src/ssr/universal/components/redirect-with-status/index.test.js

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import React from 'react'
+import {render} from '@testing-library/react'
+import {StaticRouter, Route} from 'react-router-dom'
+import {RedirectWithStatus} from './index'
+
+describe('RedirectWithStatus', () => {
+    // Dummy test needed for code coverage (the case where there is no static context)
+    test('Renders correctly', () => {
+        const targetUrl = '/target'
+        render(
+            <StaticRouter location="/redirect">
+                <Route path="/redirect">
+                    <RedirectWithStatus to={targetUrl} />
+                </Route>
+            </StaticRouter>
+        )
+    }),
```

Comment:
```suggestion
    })
```

No need for a comma here.